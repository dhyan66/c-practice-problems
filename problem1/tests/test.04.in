new 30
set 29
set 28
set 27
set 26
set 25
set 24
toggle 29
toggle 28
toggle 27
toggle 26
toggle 25
toggle 24
end
