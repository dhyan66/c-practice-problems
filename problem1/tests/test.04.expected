00 00 00 00 
00 00 00 20 
00 00 00 30 
00 00 00 38 
00 00 00 3c 
00 00 00 3e 
00 00 00 3f 
00 00 00 1f 
00 00 00 0f 
00 00 00 07 
00 00 00 03 
00 00 00 01 
00 00 00 00 
